Gemfile-Docker

```diff
@@ -0,0 +1,17 @@
+ruby '2.3.0', engine: 'jruby', engine_version: '9.1.2.0'
```

Comment:
Gemfile-Docker? 
additionally we're using 
- ruby 2.2.2p95 
- jruby 1.7.13
inside the virtualbox
Is this something can colide with that fact?